godot/src/ui/components/notifications/notification_text_helper.gd

```diff
@@ -186,7 +186,7 @@ static func get_notification_title(notif_type: String, metadata: Dictionary) ->
 				% community_name
 			)
 
-		"community_event_created":
+		"event_created":
 			var community_name = metadata.get("communityName", "Unknown Community")
 			return "The [b][%s][/b] Community has added a new event." % community_name
```

Comment:
Community is presented all the time in an event? because we also have events without a community being linked.